src/fixture.ts

```diff
@@ -185,26 +185,28 @@ export class Fixture {
         // no-op
       }
       fs.linkSync(cOut, cLink);
-      executables.push(cLink);
+      executables.push(cOut);
     }
 
     if (artifacts.js !== undefined) {
-      const jsOut = path.join(BUILD_DIR, name + '-js');
+      const jsOut = path.join(BUILD_DIR, name + `-js${process.platform === 'win32' ? '.cmd' : ''}`);
 
       const jsArgs = [
-        `-p ${path.resolve(js)}`,
+        `-p ${path.resolve(js).replace(/(\s+)/g, '\\$1')}`,
       ];
       for (const extra of extraJS) {
-        jsArgs.push(`-b ${path.resolve(extra)}`);
+        jsArgs.push(`-b ${path.resolve(extra).replace(/(\s+)/g, '\\$1')}`);
       }
 
       if (initJS) {
         jsArgs.push(`-i ${initJS}`);
       }
 
-      fs.writeFileSync(jsOut,
-        '#!/bin/sh\n' +
-        `${JS_RUNNER} ${jsArgs.join(' ')} "$1" "$2"`);
+      const content = process.platform === 'win32'
+          ? `node ${JS_RUNNER.replace(/(\s+)/g, '\\$1')} ${jsArgs.join(' ')} "%1" "%2"`
```

Comment:
Nitpick: Could you extract the `JS_RUNNER` and `jsArgs` parts of this template to a variable outside and use it here? (After reflection I think that I might have an obsession about extracting things and putting them into a variable :joy: )